pkg/handler/handler.go

```diff
@@ -141,16 +143,29 @@ func (m *Modifier) updatePodSpec(pod *corev1.Pod, roleName, audience string) []p
 		}
 	}
 
+	hostTokenFilePath := filepath.Join(m.MountPath, m.tokenName)
+	if val, ok := pod.Spec.NodeSelector["beta.kubernetes.io/os"]; ok && val == "windows" {
+		// Convert the unix file path to a windows file path
+		// Eg. /var/run/secrets/eks.amazonaws.com/serviceaccount/token to
+		//     C:\var\run\secrets\eks.amazonaws.com\serviceaccount\token
+		absolutePath, err := filepath.Abs(hostTokenFilePath)
+		if err != nil {
+			klog.Errorf("error resolving absolute path of host token file: %v", err)
```

Comment:
@micahhausler any suggestions on error handling ? Currently I'm just printing to the log and returning.

Reply:
If you remove the `Abs` call, you won't need any error handling :)